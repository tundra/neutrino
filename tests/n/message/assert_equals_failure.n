import $assert;
do $assert:equals(1, 2);
