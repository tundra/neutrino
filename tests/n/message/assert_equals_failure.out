--- backtrace ---
- leave.assert_failed()
- :fail()
- :that(false)
- :equals(1, 2)
Error: %<condition: Signal(escape)>
